docs/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # What's New with Enterprise-NG
 
+- `[Tree]` Expose unselected event. `MC` ([#https://github.com/infor-design/enterprise-ng/issues/670](https://github.com/infor-design/enterprise-ng/pull/https://github.com/infor-design/enterprise-ng/issues/670))
```

Comment:
We need to make a new 7.0.0 section in the change log. Also if you want this in 6.3.0 please PR this into the 6.3.x branch or it will go in 7.0.0 on master.

Reply:
I've made this change in the other PR, so hopefully the merge wont be too hard (one way or other).

Reply:
@micheal-cao can you please pull master branch onto this branch. Then you can place this change log comment into the new 6.3.0 section so its in the right version.